tss-esapi/src/context/tpm_commands/non_volatile_storage.rs

```diff
@@ -153,7 +154,31 @@ impl Context {
         )
     }
 
-    // Missing function: NV_Increment
+    /// Increment monotonic counter index
+    ///
+    /// # Details
```

Comment:
I know the arguments are pretty self explanatory but maybe it would be useful for consistency to add the section just like here: https://github.com/parallaxsecond/rust-tss-esapi/blob/main/tss-esapi/src/context/tpm_commands/object_commands.rs#L28 ?

Reply:
Yeah I have created a ticket to improve the documentation of all the NV context methods. Because they are quite .... there is a lot of room for improvements.

Reply:
Yes it will be fixed in another pr.